introduction.md

```diff
@@ -1 +1,4 @@
 Hi there, I am introducting myself. My favorite Code School path is JavaScript.
+* Git
+* JavaScript
+* Groovy
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?